code/parse/sexp.cpp

```diff
@@ -11981,33 +11982,43 @@ void sexp_cap_waypoint_speed(int n)
 
 /**
  * Causes a ship to jettison its cargo
- * note that the 2nd arg (jettison delay) is not implemented
  */
-void sexp_jettison_cargo(int n)
+void sexp_jettison_cargo(int n, bool jettison_new)
 {
 	char *shipname;
 	int ship_index;
+	int jettison_delay_or_speed;
```

Comment:
Yes, the delay value *isn't* actually used for `jettison-cargo-delay`, which leaves us with exactly 0 instances of this variable actually being used as an `int`, which leads us back to the question of why we're using an `int` to store a value that is only ever used as a `float`.